s3blobs/src/play/modules/s3blobs/S3Blobs.java

```diff
@@ -19,22 +19,8 @@ public void onApplicationStart() {
 		}		
 
 		Logger.info("Starting the S3Blobs module");
-		
-		if (!Play.configuration.containsKey("aws.access.key")) {
-			throw new ConfigurationException("Bad configuration for s3: no access key");
-		} else if (!Play.configuration.containsKey("aws.secret.key")) {
-			throw new ConfigurationException("Bad configuration for s3: no secret key");
-		} else if (!Play.configuration.containsKey("s3.bucket")) {
-			throw new ConfigurationException("Bad configuration for s3: no s3 bucket");
-		}
 		S3Blob.s3Bucket = Play.configuration.getProperty("s3.bucket");
-		S3Blob.serverSideEncryption = ConfigHelper.getBoolean("s3.useServerSideEncryption", false);
-		String accessKey = Play.configuration.getProperty("aws.access.key");
-		String secretKey = Play.configuration.getProperty("aws.secret.key");
-		AWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
-		S3Blob.s3Client = new AmazonS3Client(awsCredentials);
-		if (!S3Blob.s3Client.doesBucketExist(S3Blob.s3Bucket)) {
-			S3Blob.s3Client.createBucket(S3Blob.s3Bucket);
-		}
+		S3Blob.serverSideEncryption = ConfigHelper.getBoolean("s3.useServerSideEncryption", true);
```

Comment:
is this safe to now default to true? How does it affect existing bucket items?

Reply:
it will be applied for new objects. it will be all seamless and by default server-side encryption is enabled in terraform.